docs/README.md

```diff
@@ -31,15 +31,41 @@ whichever version of Spark you currently have checked out of revision control.
 The Spark documentation build uses a number of tools to build HTML docs and API docs in Scala, Java,
 Python, R and SQL.
 
-You need to have [Ruby](https://www.ruby-lang.org/en/documentation/installation/) and
-[Python](https://docs.python.org/2/using/unix.html#getting-and-installing-the-latest-version-of-python)
-installed. Also install the following libraries:
+You need to have Ruby 2 and Python 3 installed. A handy way to install and manage various versions of Ruby and Python is with [`rbenv`] and [`pyenv`].
```

Comment:
Can we revert these doc changes in the documentation? Otherwise, I am good with other changes.

Reply:
Or alternatively, we can have one section to describe use `rbenv` and `pyenv` separately .. that works to me too.

Reply:
I moved them to their own sub-section. If we think it's distracting / not helpful, I can remove it.